includes/class-freemius.php

```diff
@@ -4495,9 +4495,9 @@ function dynamic_init( array $plugin_info ) {
             }
 
             if ( $this->has_api_connectivity() ) {
-                if ( self::is_cron() ) {
-                    $this->hook_callback_to_sync_cron();
-                } else if ( $this->is_user_in_admin() ) {
+                $this->hook_callback_to_sync_cron();
+
+                if ( $this->is_user_in_admin() ) {

```

Comment:
Let's do this just to be safe about the logic change

```suggestion
                if ( ! self::is_cron() && $this->is_user_in_admin() ) {

```